schemas/com.getvero/bounced/jsonschema/1-0-0

```diff
@@ -12,25 +12,60 @@
 	"properties": {
 		"bounced_at": {
 			"type": "string",				
-			"format": "date-time"
+			"format": "date-time",
+			"maxLength": 10
 		},
 		"bounce_type": {
-			"type": "string",
+			"type": ["string", "null"],
+			"maxLength": 255
+		},
+		"bounce_code": {
+			"type": ["string", "null"],
+			"maxLength": 32
+		},
+		"bounce_message": {
+			"type": ["string", "null"],
+			"maxLength": 255
+		},
+		"message_id": {
+			"type": ["string", "null"],
 			"maxLength": 255
 		},
 		"type": {
 			"type": "string",
 			"maxLength": 12
 		},
+		"event": {
+			"type": "object",
+			"properties": {
+				"name": {
+					"type": ["string", "null"],
+					"maxLength": 255
+				},
+				"triggered_at": {
+					"type": ["string", "null"],
+					"format": "date-time"
+				},
+				"data": {
+					"type": "object",
+					"properties": {
+						"key": {
```

Comment:
`data` is most likely meant to be a dictionary-like object that can store arbitrary key/value pairs. From Vero's example event:

`"data":{"key":"value"}`

I'd just set `"additionalProperties": true` here with no required properties. (Also applies to `event.data` fields in other schemas)

Reply:
If you're aware of content of keys or values, you can also use `patternProperties` to restrict them to specific types.

Reply:
Agree with all that